### DISCUSSION TITLE
The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing in 2.3.0

### DISCUSSION THREAD

**Discussion by rairafaqatali** [elsa-workflows/elsa-core#1586] [Q&A] [answered]
Hi @sfmskywalker 
I have tried to set timeout with below way, but none of them worked. :
 `.AddHttpActivities(elsaSection.GetSection("Server").Bind, configureHttpClient: httpClientBuilder => httpClientBuilder.SetHandlerLifetime(TimeSpan.FromHours(10)))`

` .AddHttpActivities(elsaSection.GetSection("Server").Bind, configureHttpClient: httpClientBuilder => httpClientBuilder.ConfigureHttpClient(x => x.Timeout = TimeSpan.FromDays(1)))`

When I use send HTTP request activity, it throws **The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.**
Can you let me know what I'm missing?
 Is this issue resolved or still pending?

Kindly let me know.

Thanks

**Comment by sfmskywalker** [comment, marked as answer]
Hi @rairafaqatali , I think you should set the timeout like this:

```csharp
.AddHttpActivities(configureHttpClient: builder => builder.ConfigureHttpClient(client => client.Timeout = TimeSpan.FromDays(1)))
```

**Reply by rairafaqatali** [reply]
@sfmskywalker 
It's not working 
`.AddHttpActivities(configureHttpClient: builder => builder.ConfigureHttpClient(client => client.Timeout = TimeSpan.FromDays(1)))`

**Reply by sfmskywalker** [reply]
Are you sure it’s the client that’s timing out and not the server?
